src/scripts/utils.py

```diff
@@ -167,6 +170,18 @@ def _load_ontology(ontology_path: str, use_cache=False) -> ProntoImplementation:
     return ontology
 
 
+def get_monarch_curies_converter(from_prefixes_csv: Union[str, Path] = DEFAULT_PREFIXES_CSV) -> curies.Converter:
```

Comment:
## `get_monarch_curies_converter()`
It seems to me this is something we will re-use in our Python scripts, so I created a general function for it. Basically, this returns a `curies.Converter` that loads our prefix map.

Implementation: I did CSV -> YAML -> Dict. I can change to CSV -> Dict directly for simplicity. Just wanted to see what it would look like if we wanted to convert `prefixes.csv` to YAML.

Reply:
I dont trust Monarch converter, as we don't maintain it. Lets use the `converter` provided by SSSOM. Can you reach out to Harshad and ask him how to just re-use the SSSOM converter? This is based on the obocontext, and this one we do curate.

Reply:
Very nice initiative though to factoring out this code 100% agree.  Also you have now seen Charlies proposal on cure remapping and rewiring, maybe this is a good time to try this (load sssom converter (which is a curies converter), and then remap according to what is in the `prefixes.csv`).

Reply:
@hrshdhgd Can you follow up with me on here or Slack about the SSSOM curie `converter` Nico mentioned above?

@matentzn Yeah that sounds like a good idea. I saw he merged and it looks like he just [released](https://github.com/cthoyt/curies/releases/tag/v0.6.3) too. Just to be clear on what you mean, you want me to use the remap/rewiring on the SSSOM `converter`, right? I'm guessing/hoping that will work like this: (i) I load the SSSOM converter in to memory, (ii) I update it using remap/rewire and `prefixes.csv`, (iii) I re-export it or something so that the next time I load it, it's up-to-date.

Then the question is... every time I want to update a prefix map, I'm supposing I we ditch `prefixes.csv` and do this in the `sssom` repo, and then some how re-import into `mondo-ingest`. Also, OAK needs the prefixes in CSV form, so we need something like a prefixes YAML -> CSV converter in `mondo-utils`.

Maybe best to discuss this at meeting.

Reply:
Not an issue currently for this PR.

Follow-up issue created:
- https://github.com/monarch-initiative/mondolib/issues/8